plugins/caches.js

```diff
@@ -150,6 +150,48 @@ exports.plugin = {
                     }
                 }
             }
+        }, {
+            method: 'DELETE',
+            path: '/events/{id}/{cacheName}',
+            handler: async (request, h) => {
+                const { eventId } = request.auth.credentials;
+                const eventIdParam = request.params.id;
+
+                if (eventIdParam !== eventId) {
```

Comment:
From this code, only the same event has permission to delete the cache. I think eventually we need to extend this to an `admin` (admin of the pipeline maybe?) to have the permission to invalidate the cache.